adflow/pyADflow.py

```diff
@@ -3738,6 +3738,8 @@ def computeJacobianVectorProductFwd(
         funcDeriv=False,
         fDeriv=False,
         groupName=None,
+        mode="AD",
```

Comment:
Can you add the explanation of mode and h to the doctoring? It would help if you explain what the options are for mode, and how they all work. 

Also, I am conflicted by the default value for h; its only good for FD, its not used in AD and should be modified for CS. Is there a better way we can specify this? Maybe have none as default and each case can overwrite its own default unless the user specifies the value they want?

Reply:
I changed the default value of h to None